apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,29 @@
+import logoSrc from "~/assets/images/logo.svg";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+import { Avatar } from "../components.js";
+import styles from "./styles.module.css";
+
+const Header = (): JSX.Element => {
+	const authenticatedUser = useAppSelector(
+		({ auth }) => auth.authenticatedUser,
+	);
+
+	if (!authenticatedUser) {
+		return <></>;
+	}
+
+	const userName = authenticatedUser.user.name;
```

Comment:
Hm, authenticatedUser should already be a user with name property, isn't it?

Reply:
Auth slice currently looks like this:
![image](https://github.com/user-attachments/assets/adac688b-02a8-4b17-aba7-2979c461b26c)

Reply:
ok, we will change this in a separate ticket